examples/python/example_type_awareness.py

```diff
@@ -0,0 +1,141 @@
+    Column('token_str', Text, nullable=True),
+    Column('token_bytes', LargeBinary, nullable=True),
+    Column('token_email', Text, nullable=True),
+)
+# _schema_test_table used to generate table in the database with binary column types
+_schema_test_table = Table(
+    'test', metadata,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('data_str', LargeBinary, nullable=True),
+    Column('masking', LargeBinary, nullable=True),
+    Column('token_i32', Integer, nullable=True),
+    Column('data_i32', LargeBinary, nullable=True),
+    Column('token_i64', BigInteger, nullable=True),
+    Column('data_i64', LargeBinary, nullable=True),
+    Column('token_str', Text, nullable=True),
+    Column('token_bytes', LargeBinary, nullable=True),
+    Column('token_email', Text, nullable=True),
+)
+
+rotation_test_table = Table(
+    'users', metadata,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('email', LargeBinary, nullable=True),
+)
+
+table_map = {
+    test_table.name: test_table,
+    rotation_test_table.name: rotation_test_table,
+}
+
+
+def print_data(connection, columns, table=test_table):
+    """fetch data from database and print to console"""
+    default_columns = ['id']
+    try:
+        if columns:
+            table_columns = [table.c.id] + [
+                getattr(table.c, i) for i in columns]
+            query = select(table_columns)
+            extra_columns = columns
+        else:
+            table_columns = [table.c.id] + [
+                i for i in table.columns if i.name not in default_columns]
+            query = select(table_columns)
+            extra_columns = [i.name for i in table.columns if i.name not in default_columns]
+    except AttributeError:
+        print("\n\n{0}\nprobably you used incorrect column name\n{0}\n\n".format('*' * 30))
+        raise
+        exit(1)
+
+    print("Fetch data by query {}\n",
+          query.compile(dialect=postgresql.dialect(), compile_kwargs={"literal_binds": True}))
+    result = connection.execute(query)
+    result = result.fetchall()
+
+    print(len(result))
+    print("{:<3} - {}".format(*default_columns, ' - '.join(extra_columns)))
+    for row in result:
+        values = ['{:<3}'.format(row['id'])]
+        for col in row[1:]:
+            if isinstance(col, (bytes, bytearray)):
+                values.append(col.decode('utf-8', errors='ignore'))
+            else:
+                values.append(str(col))
+
+        print(' - '.join(values))
+
+
+def write_data(data, connection, table=test_table):
+    # here we encrypt our data and wrap into AcraStruct
+    with open(data, 'r') as f:
+        data = json.load(f)
+    print("data: {}".format(data))
+    rows = data
+    if isinstance(data, dict):
+        rows = [data]
+    for row in rows:
+        # explicitly encode bytes data to bytes so alchemy send it as hexadecimal string in insert query
+        row['token_bytes'] = row['token_bytes'].encode('ascii')
```

Comment:
let's check that row contain this field. Most of fields are nullable and someone may pass data without specific column and we will get KeyError for valid case